src/fe/fe_abstract.C

```diff
@@ -888,6 +892,14 @@ void FEAbstract::compute_node_constraints (NodeConstraints & constraints,
         elem->neighbor_ptr(s) != remote_elem)
       if (elem->neighbor_ptr(s)->level() < elem->level()) // constrain dofs shared between
         {                                                 // this element and ones coarser
+
+#ifdef LIBMESH_ENABLE_INFINITE_ELEMENTS
+           // for infinite elements, only the base is constrained.
```

Comment:
Is this really correct?  I'd have been certain you need constraints on the sides with neighboring infinite elements too.  Although I admit I wouldn't have the slightest idea how to calculate them.

Reply:
I am not absolutely certain; but since there is no splitting in radial direction, I don't see how the basis could get discontinuous as long as the basis is not.

Reply:
Are there more DoFs in the radial direction, though?  That's all you'd need for inconsistency in 3D.  If you have a coarse hex with two fine hexes next to it, all three of which have infinite neighbors, then just because the DoFs on the hanging node at the finite->infinite boundary are properly constrained doesn't mean the DoFs on the extended node (the one "closer to infinity" in cell_inf_hex8.h parlance, whose edge in that diagram leads to the hanging node) will be properly constrained too.

Reply:
I see. You are right; this must be done 'somehow'...